docs/intro-to-coalton.md

```diff
@@ -546,6 +546,8 @@ All of these cases are sufficiently common that we provide a few shorthands:
 
 - `floor/`, `ceiling/`, and `round/` for integer-to-integer division, and
 
+- `fromfrac` for converting a `Fraction` into a type with a `Dividable Integer :a` instance (this conversion may result in precision loss)
```

Comment:
I think this deserves an example, and maybe shouldn't be a bullet point, but be a separate paragraph altogether.